src/components/pages/footer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import './Footer.css'
+
+function Footer () {
+    return (
+        <footer className="footer"> 
+        <hr className="bar"/>
+        Made with Love for FOSS in India
+        </footer>
```

Comment:
Seems like this has missed the autoformat, 
can you once execute `npm run format`?